### PR TITLE
Use new-style issue templates

### PR DESCRIPTION
**Pull request by maya-r**
Taken from kubevirt/kubevirt#7416, adjusted to refer to CDI.

Co-Authored-By: Ben Oukhanov <boukhano@redhat.com>
Signed-off-by: Ben Oukhanov <boukhano@redhat.com>
Signed-off-by: Maya Rashish <mrashish@redhat.com>

**What this PR does / why we need it**:
From the kubevirt PR:
> This is an update for the issue templates, KubeVirt is using an old version of issue templates (see the warning [here](https://github.com/benukhanov/kubevirt/blob/main/.github/ISSUE_TEMPLATE.md) and our [Community Standards](https://github.com/kubevirt/kubevirt/community)).
> 
> Label `kind/bug` or `kind/enhancement` will be set automatically. The user won't need to write it. smile
> 
> Before opening a new issue the user will be able to choose a template:
> 
>     1. Bug Report
> 
>     2. Docs Report
> 
>     3. Feature Request
> 
> 
> Please see an example [here](https://github.com/kubernetes/kubernetes/issues/new/choose).

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+**How to reproduce it (as minimally and precisely as possible)**:
+Steps to reproduce the behavior.
+
+**Additional context**:
+Add any other context about the problem here.
+
+**Environment**:
+- CDI version (use `kubectl get deployments cdi-deployment -o yaml`): N/A
+- Kubernetes version (use `kubectl version`): N/A
+- VM or VMI specifications: N/A
```
Comment (awels) [inline .github/ISSUE_TEMPLATE/bug_report.md:27]: Maybe change this to DV specification?

**Review by awels — COMMENTED** [review]
Looks good one minor comment

**Comment by awels** [comment]
/override pull-containerized-data-importer-e2e-k8s-1.22-upg
/override pull-containerized-data-importer-e2e-k8s-1.22-hpp-istio

**Comment by brybacki** [comment]
test 7414 failed, the Update should be done in a loop,

**Pull request merged**